### ISSUE TITLE
Clavicle sparing hypomineralization

### ISSUE THREAD

**Issue by poratshay** [obophenotype/human-phenotype-ontology#11566]
**Preferred term label:**
Clavicle-sparing-hypomineralization

**Synonyms**

**Definition (free text, please give PubMed ID)**
 It characterizes the skeletal pattern seen in hypophosphatasia, where despite widespread hypomineralization affecting the skull, ribs, and long bones, the clavicles show relative preservation of mineralization compared to other skeletal elements. PMID 12424591

**Parent term (use hpo.jax.org/app)**
 ORPHA:436

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 ORPHA:436 

**Your nano-attribution (ORCID)**
0000-0001-9947-035X

**Comment by pnrobinson** [comment]
@azankl is there a standard definition for this feature?

**Comment by azankl** [comment]
@pnrobinson I am not aware of this being a consistent feature of hypophosphatasia. The reference given above does not comment on this feature. Would only consider this term if a better reference is provided.

**Comment by pnrobinson** [comment]
@poratshay @azankl 
Thanks Andreas.
I agree this is not an actual entity. In Figure three of this paper, it is stated "clavicles are absent ", and so there is no "clavicle-sparing". If we really needed to annotate this, then multiple terms for each of the regions could be used!
